resources/js/Pages/Layout.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <main>
+    <header>
+      <img src="../../img/wikidata-logo.svg" alt="Wikidata-logo" /> 
```

Comment:
Same as above
```suggestion
      <img src="images/wikidata-logo.svg" alt="Wikidata-logo" /> 
```

Reply:
Wait, no, I'm a bit of a dunce! Now I see why you were using the source image and the webpack loader. -> for asset versioning. Resolving!